README.md

```diff
@@ -1,34 +1,43 @@
 ![](./resources/official_armmbed_example_badge.png)
 # Running Mbed Crypto examples on Mbed OS
-This repository contains a set of examples demonstrating the compilation and use of Mbed Crypto on Mbed OS.
-
-List of examples contained within this repository:
-* Example code snippets for using the library, with [documentation](https://github.com/ARMmbed/mbed-crypto/blob/development/docs/getting_started.md).
+This repository contains an example demonstrating the compilation and use of Mbed Crypto on Mbed OS.
```

Comment:
Existing issue, but we should say "PSA Crypto" instead of "Mbed Crypto". The example is expected to work with any PSA crypto implementation.

Reply:
Good point. Shall we also rename this repo to mbed-os-example-psa-crypto? Existing links to the old names will be redirected by GitHub.